examples/go/tagging/handler/handler.go

```diff
@@ -77,13 +75,13 @@ func Handle(req handler.Request) (handler.Response, error) {
 
 	// Fill information in the request into a govmomi type.
 	var moRef types.ManagedObjectReference
-	moRef.Value = mor.Data.Vm.Vm.Value
-	moRef.Type = mor.Data.Vm.Vm.Type
+	moRef.Value = event.Event.Vm.Vm.Value
+	moRef.Type = event.Event.Vm.Vm.Type
 
 	// Connect to vcenter with govmomi client
-	log.Println("Connecting to vCenter")
+	log.Println("connect to vCenter")
 	clients := connect(ctx, vcc.VCenter)
```

Comment:
Assuming some of the earlier comments on client handling are still not incorporated (e.g. error handling), so skipping this one